docs/source/algorithms/PDFFourierTransform-v2.rst

```diff
@@ -0,0 +1,189 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The algorithm transforms a single spectrum workspace containing 
+spectral density :math:`S(Q)`, :math:`S(Q)-1`, or :math:`Q[S(Q)-1]` 
+(as a function of **MomentumTransfer** or **dSpacing** :ref:`units <Unit Factory>`) to a PDF 
+(pair distribution function) as described below and also the reverse. The available PDF types are the
+reduced pair distribution function :math:`G(r)`, the pair distribution function :math:`g(r)`, and the
+radial distribution function :math:`RDF(r)`.
+
+The output from this algorithm will have an x-range between 0.0 and the maximum parameter of the output,
+i.e. if converting from `g(r)` to `S(Q)` the output will be between 0.0 and `Qmax`.
+
+The spectrum density should be in the Q-space (\ **MomentumTransfer**\ ) :ref:`units <Unit Factory>` .
+(d-spacing is not supported any more. Contact development team to fix that and enable **dSpacing** again)
+
+References
+----------
+
+#. B. H. Toby and T. Egami, *Accuracy of Pair Distribution Functions Analysis Appliced to Crystalline and Non-Crystalline Materials*, Acta Cryst. (1992) A **48**, 336-346
+   `doi: 10.1107/S0108767391011327 <http://dx.doi.org/10.1107/S0108767391011327>`_
+#. B.H. Toby and S. Billinge, *Determination of Standard uncertainties in fits to pair distribution functions*  Acta Cryst. (2004) A **60**, 315-317]
+   `doi: 10.1107/S0108767304011754 <http://dx.doi.org/10.1107/S0108767304011754>`_
+
+.. The algorithm itself is able to identify the unit.  -- not any more. TODO:  should be investigated why this has been disabled
+
+
+PDF Options
+--------------
+
+**G(r)**
+########
+
+.. raw:: html
+
+   <center>
+
+:math:`G(r) = 4\pi r[\rho(r)-\rho_0] = \frac{2}{\pi} \int_{0}^{\infty} Q[S(Q)-1]\sin(Qr)dQ`
+
+.. raw:: html
+
+   </center>
+
+and in this algorithm, it is implemented as
+
+.. raw:: html
+
+   <center>
+
+:math:`G(r) =  \frac{2}{\pi r^2} \sum_{Q_{min}}^{Q_{max}} (S(Q)-1)[\sin(Qr) - Qr\cos(Qr)]^{right bin}_{left bin}`
```

Comment:
Please could you add M(Q,Q_max) into this expression so that the following comment makes sense ("where M(Q,Q_max) is an optional filter function...")